tests/kafkatest/services/security/security_config.py

```diff
@@ -276,18 +287,19 @@ def setup_node(self, node):
         if java_version(node) <= 11 and self.properties.get('tls.version') == 'TLSv1.3':
             self.properties.update({'tls.version': 'TLSv1.2'})
 
-    def setup_credentials(self, node, path, zk_connect, broker):
-        if broker:
-            self.maybe_create_scram_credentials(node, zk_connect, path, self.interbroker_sasl_mechanism,
+    def setup_credentials(self, node, path, connect, creating_broker_user):
```

Comment:
I think we should just have a separate function for this, since there's no code shared between when creating_broker_user=True and when it is False.